### ISSUE TITLE
Error "Failed to resolve import "ohmyfetch"" after upgrade to Nuxt 3.1.0

### ISSUE THREAD

**Issue by silvio-e** [nuxt-modules/sanity#644]
### Version
module: 1.5.0
nuxt: 3.1.0

## Reproduction

https://stackblitz.com/edit/nuxt-starter-kfrrf1

#### What is expected?

Nuxt with the sanity module starts without problems.

#### What is actually happening?

```
ERROR  Failed to resolve import "ohmyfetch" from "node_modules/@nuxtjs/sanity/dist/runtime/client.mjs?v=451a1d82". Does the file exist? 
```

#### Steps to reproduce

npm run dev

## Additional information

The only thing I added is using `useSanityQuery` in `app.vue` and then the error appears.

## Checklist
* [x]  I have tested with the latest Nuxt version and the issue still occurs
* [x]  I have tested with the latest module version and the issue still occurs
* [x]  I have searched the issue tracker and this issue hasn't been reported yet

**Comment by danielroe** [comment]
Thanks for your quick spot! This should be resolved in v1.6.0.

**Comment by silvio-e** [comment]
Thank you for fixing it so quickly! 😃
